clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -70,6 +70,16 @@ Code completion
 Code actions
 ^^^^^^^^^^^^
 
+- New ``clangd`` code action: "Override pure virtual methods". When invoked on a
+  class definition, this action automatically generates C++ ``override``
+  declarations for all pure virtual methods inherited from its base classes that
+  have not yet been implemented. The generated method stubs include a ``TODO``
```

Comment:
i think content starting with `The generated method stubs include a ``TODO`` ...` is too much details for release notes, i'd probably drop.